help-center/faq/cloud-faq.md

```diff
@@ -0,0 +1,83 @@
+---
+id: cloud-faq
+title: Memgraph Cloud - Frequently asked questions
+sidebar_label: Memgraph Cloud
+slug: /faq/cloud
+---
+
+## Pricing and payment
+
+### What is the pricing?
+
+You can estimate the cost of Memgraph Cloud's service by selecting your cloud
+region and instance size with our [Cost
+Calculator](https://cloud.memgraph.com/pricing), or you can check them out at
+[Project rates](/docs/memgraph-cloud/payment#project-rates))
+
+### How can I use a coupon that I got for Memgraph Cloud?
+First, you need to add a credit card to your account. Then you can apply the
+coupon.
```

Comment:
Change the title: How can I redeem a coupon...
Change the word apply in the body, with the word redeem

link to https://memgraph.com/docs/memgraph-cloud/payment#add-a-payment-method on the words "add a credit card"
link to https://memgraph.com/docs/memgraph-cloud/payment#add-a-payment-method on the words redeem the coupon